client/src/app/site/pages/meetings/pages/interaction/services/interaction.service.ts

```diff
@@ -78,6 +80,8 @@ export class InteractionService {
         private promptService: PromptService,
         private broadcast: BroadcastService
     ) {
+        InteractionServiceRegistry = InteractionServiceRegistry.concat(this);
```

Comment:
Mmh, I don't particularily like this solution (it seems like the underlying problem is somewhere in the DI and this is just symptom fixing), but it's fine by me if it works.

Reply:
This problem indeed lies with the DI. Mainly it is because this service is not provided in `root` and a new instance of it is therefore created whenever it is injected into a new module. I _tried_ fixing this by changing where it is provided but failed due to its dependencies. I _could_ try to create a new `root` service for the purpose of handling received messages, but there is no guarantee that this would work either.